src/parkapi_sources/converters/kienzler/converter.py

```diff
@@ -9,36 +9,59 @@
 from validataclass.exceptions import ValidationError
 from validataclass.validators import AnythingValidator, DataclassValidator, ListValidator
 
-from parkapi_sources.converters.base_converter.pull import PullConverter
+from parkapi_sources.converters.base_converter.pull import PullConverter, StaticGeojsonDataMixin
 from parkapi_sources.exceptions import ImportParkingSiteException
 from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
 
-from .models import KienzlerInput
+from .models import KienzlerGeojsonFeatureInput, KienzlerInput
 
 
-class KienzlerBasePullConverter(PullConverter):
+class KienzlerBasePullConverter(PullConverter, StaticGeojsonDataMixin):
     kienzler_list_validator = ListValidator(AnythingValidator(allowed_types=[dict]))
     kienzler_item_validator = DataclassValidator(KienzlerInput)
+    kienzler_geojson_item_validator = DataclassValidator(KienzlerInput)
+    use_geojson = False
 
     @property
     @abstractmethod
     def config_prefix(self):
         pass
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, use_geojson=False, **kwargs):
```

Comment:
Missing typing. But: you cannot set this attribute anyway. Instead, it should be set at the class definition as you already do above.

Now it's also clear which subclasses should set `se_geojson = True`. It's Offenburg, RadSafe, Stuttgart and VRN (see https://github.com/ParkenDD/parkapi-static-data/pull/6/files ).